src/io.c

```diff
@@ -1102,6 +1150,11 @@ static void PutLine2(TypOutputFile * output, const Char * line, UInt len)
     }
     else {
         SyFputs(line, output->file);
+        if (output->file == 1 || output->file == 3) {
+            // FIXME: this does not work right for unicode output,
+            // but we don't handle this right anyway, so...
+            output->column += output->pos;
```

Comment:
I guess I could (should?) drop the stdout/errout check...

also, this (although it has no effect in practice, but it may seem less surprising to anybody reading the code)
```suggestion
            output->column += len;
```